Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -70,8 +70,23 @@ namespace AZ
             };
             CullData m_cullData;
 
+            enum LodType : uint8_t
+            {
+                Default = 0,
+                ScreenCoverage,
+                SpecificLod,
+            };
             using LodOverride = uint8_t;
-            static constexpr uint8_t NoLodOverride = AZStd::numeric_limits<LodOverride>::max();
+
+            struct LodConfiguration
+            {
+                LodType m_lodType = LodType::Default;
```

Comment:
I have a slight preference to putting the smaller types at the end, although I know it won't change the struct size and having LodType first kind of reads better. Up to you if you want to change this one.

Reply:
I think I want to keep it the same, I like it when things are declared above where they're used.